src/zcl_abapgit_transport_mass.clas.locals_imp.abap

```diff
@@ -0,0 +1,341 @@
+********************************************************************************
+* The MIT License (MIT)
+*
+* Copyright (c) 2019 abapGit Contributors
+*
+* Permission is hereby granted, free of charge, to any person obtaining a copy
+* of this software and associated documentation files (the "Software"), to deal
+* in the Software without restriction, including without limitation the rights
+* to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+* copies of the Software, and to permit persons to whom the Software is
+* furnished to do so, subject to the following conditions:
+*
+* The above copyright notice and this permission notice shall be included in all
+* copies or substantial portions of the Software.
+*
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+* IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+* AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+* LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+* OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+* SOFTWARE.
+********************************************************************************
+*( )_( )
+*(='.'=)
+*(")_(")
+* This program allow to generate Abapgit ZIP files from transport request(s)
+* in a given folder with the given logic ( FULL or PREFIX )
+
+*================================ CLASSES =============================*
+
+*================================  GUI ================================*
+CLASS lcl_gui DEFINITION FINAL.
```

Comment:
later refactoring: suggest moving some of these methods to ZCL_ABAPGIT_POPUPS so they can be reused from there, but lets keep as is for now

Reply:
OK. But to save some coding lines I created the method static class save_binstring_to_localfile in zcl_abapgit_zip in order to reuse it in lcl_transport_zipper->my generate_files method as well as in zcl_abapgit_zip=>export_object . It gets the code cleaner as well imho.